gateway/src/integration-test/resources/http/grafana/query/extract-algo/testMissingAdd/testInvalidPointsSize/testAdd/expectedResponse.json

```diff
@@ -0,0 +1 @@
+{"status":"OK","message":"injected 3 points of 2 metrics in 2 chunks", "errors": "can't add this point [1477895624866] of this object {\"name\":\"openSpaceSensors.Temperature111\",\"points\":[[1477895624866],[1477916224866,4.0]]}\n"}
```

Comment:
Ok just make errors a JsonArray not just a string.

Reply:
done